src/anaconda_auth/cli.py

```diff
@@ -332,3 +341,247 @@ def auth_logout(at: Optional[str] = None) -> None:
     """Logout"""
     _override_default_site(at)
     logout()
+
+
+sites_app = typer.Typer(
+    name="sites",
+    add_completion=False,
+    help="Manage your Anaconda site configuration",
+    context_settings={
+        "allow_extra_args": True,
+        "ignore_unknown_options": True,
+        "help_option_names": ["--help", "-h"],
+    },
+)
+
+
+@sites_app.command(name="list")
+def sites_list() -> None:
+    """List configured sites by name and domain."""
+    sites_config = AnacondaAuthSitesConfig()
+
+    table = Table("Site name", "Domain name", "Default site", header_style="bold green")
+
+    for name, site in sites_config.sites.items():
+        is_default = CHECK_MARK if name == sites_config.default_site else ""
+        table.add_row(name, site.domain, is_default)
+
+    console.print(table)
```

Comment:
It might be nice to also include a prompt like "to view site details, use `anaconda sites show`. Or something like that.